.formatter.exs

```diff
@@ -5,11 +5,10 @@
     "mix.exs",
     "apps/*/mix.exs",
     "apps/*/{lib,test,config}/**/*.{ex,exs}",
-    "priv/config/config.exs",
-    "priv/mix.exs",
-    "priv/features/contexts/*.exs",
```

Comment:
I don't think this `features` folder is still valid, can you confirm I am not missing anything when removing this? @InoMurko

Reply:
you're right!

Reply:
you're correct!